frontend/pages/[gameId]/[playerId].tsx

```diff
@@ -0,0 +1,57 @@
+import { WEBSOCKET_BASE_URL } from '/Users/gealoro/kakaw/frontend/lib/api';
+import { useRouter } from 'next/router'
+import Link from 'next/link';
+import React, { useState, useCallback, useEffect } from 'react';
+import useWebSocket, { ReadyState } from 'react-use-websocket';
+
+//import './App.css';
+const WS_URL = WEBSOCKET_BASE_URL
+
+/*/
+function App() {
+useWebSocket(WS_URL, {
+    onOpen: () => {
+      console.log('WebSocket connection established.');
+    }
+  });
+
+  return (
+    <div>Hello WebSockets!</div>
+  );
+}
+/*/
+
+function ConnectPage() {
+
+  useWebSocket(WS_URL, {
```

Comment:
```suggestion
	const router = useRouter();
	const { gameId, playerId } = router.query as {
		gameId: string;
		playerId: string;
	};

	const url = new URL('/connect', WEBSOCKET_BASE_URL);
	url.searchParams.set('gameId', gameId);
	url.searchParams.set('playerId', playerId);

	useWebSocket(url.href, {
```
`WEBSOCKET_BASE_URL` is just the hostname/port we should connect to. So right now it's `ws://localhost:8080` but maybe later we would change it to `wss://kakaw.com` or wherever we might end up hosting this app. You still need to specify `/connect` as well as the query parameters for the game and player IDs.